astropy/wcs/tests/test_keyedsip.py

```diff
@@ -0,0 +1,17 @@
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from ... import wcs
+
+def read_keyed_wcs():
+    w = wcs.WCS( header = "data/sip_a_only.hdr", key = 'A' )
+    return( w )
```

Comment:
Get the path to the file using

`path = get_pkg_data_filename("data/sip_a_only.hdr")`
Isn't there a file in the tests/data directory that can be used for this purpose? I'd prefer not to put new files in the repository.
Add reading the file and initializing the WCS to the test function. You can look for an example in test_wcs.py.